.gitignore

```diff
@@ -1,3 +1,8 @@
-csv/.gradle/
-csv/build/
-*.class
+# Gradle
```

Comment:
This needs merging with the existing gitignore

Reply:
Logically it supersedes the existing one. Or did I miss some files?

Reply:
Is the csv folder gone? I think its still there...

Reply:
Entries in `.gitignore` that don't start with a `/` are applied to every sub-folder.